src/signals/incident/components/form/ContainerSelect/Selector/Selector.tsx

```diff
@@ -78,6 +78,12 @@ const StyledMap = styled(Map)`
   }
 `;
 
+const StyledMapPanelDrawer = styled(MapPanelDrawer)`
+  & > :first-child {
```

Comment:
This could just as well have been changed in the amsterdam-react-maps library (and then be more resilient). Why not create a PR there?